examples/gcoap/server.c

```diff
@@ -60,11 +60,17 @@ static ssize_t _encode_link(const coap_resource_t *resource, char *buf,
                             size_t maxlen, coap_link_encoder_ctx_t *context);
 static ssize_t _stats_handler(coap_pkt_t* pdu, uint8_t *buf, size_t len, coap_request_ctx_t *ctx);
 static ssize_t _riot_board_handler(coap_pkt_t* pdu, uint8_t *buf, size_t len, coap_request_ctx_t *ctx);
+#if IS_USED(MODULE_GCOAP_DTLS)
```

Comment:
Another thing: I believe there's value in having the example code of how to use this feature, but seeing how much the example app is growing, do you think you can find a good spot in the docs to place it instead?

Reply:
Sure, I can move the code somewhere else and maybe put a bit of additional explanation text. Should this go into the readme of the example or somewhere with CoAP, DTLS or DSM?

Reply:
Hmm I think adding some section to gcoaps doc would be enough

Reply:
Or maybe add another example app?
There is a lot of value in being able to compile the code on CI, if it's just hidden in the docs somewhere it might not even more anymore after some time without anyone noticing.

Reply:
Later on this functionality will be used inside the example that I'm writing for the ACE DTLS implementation. Would that be enough (maybe + some addition in the doc?) or is a dedicated example better?